tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/species_features.tsx

```diff
@@ -71,6 +71,10 @@ export const underwear_color: Feature<string> = {
   component: FeatureColorInput,
 };
 
+export const feature_pod_hair: FeatureChoiced = {
```

Comment:
this isn't supposed to be a thing if you want it to be like other features where you have a bunch of icons to choose from, like hair.